matplotlibrc

```diff
@@ -0,0 +1,3 @@
+#### MATPLOTLIBRC FORMAT
+
+backend : Agg
```

Comment:
Can you move this to `tests` and then have the travis setup script copy it from `tests` to the working directory? I think it might be a little confusing to have this file in the root of the repo.

Reply:
done